include/entity.h

```diff
@@ -2113,17 +2113,21 @@ typedef struct {
 } ET_Skelerang;
 
 typedef struct {
-    /* 0x7C */ s32 : 32;
-    /* 0x80 */ s32 : 32;
+    /* 0x7C */ struct Entity* parent;
+    /* 0x80 */ struct Entity* entity;
```

Comment:
I think these are actually references to different parts of Diplo but I'll come back and rename these later once I have a handle on it